docker/Dockerfile.agent.multiarch

```diff
@@ -15,6 +16,7 @@ EXPOSE 3000
 COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 # copy agent binary
 COPY --from=build /src/dist/woodpecker-agent /bin/
+COPY --from=build /etc/woodpecker /etc
```

Comment:
```suggestion
RUN mkdir -p /etc/woodpecker
```

Reply:
It's not possible to run mkdir in a scratch image since there is not even a BusyBox binary. To be consistent, I did the same in the alpine Dockerfile.